README.md

```diff
@@ -50,8 +50,8 @@ Prerequisites:
 
 * Linux (if you are not on Linux, you will need to either set up a VM with the
   proper environment or, if Docker is available for your platform, use the
-  provided Docker image which does this for you, [see below](
-  #using-the-development-docker-image)).
+  provided Docker image which does this for you,
+  [see below](#using-the-development-docker-image)).
```

Comment:
I'm curious if this is just visual/human issue or does any tool have problems with that?

Reply:
my IDE's shabby markdown mode ):

Reply:
![image](https://user-images.githubusercontent.com/38229038/75186120-2d4d2800-56fc-11ea-87b6-dcc45f2ef06e.png)

Reply:
Aha, I see :slightly_smiling_face:.

No worries, we can wrap it like you suggested.